tests/foreman/cli/test_contentviews.py

```diff
@@ -315,6 +318,180 @@ def test_delete_cv_version(self):
         result = ContentView.version_delete({u'id': version1_id})
         self.assertEqual(result.return_code, 0)
 
+    def test_remove_cv_environment(self):
+        """@Test: Remove content view from lifecycle environment assignment
+
+        @Feature: Content Views
+
+        @Assert: Content view removed from environment successfully
+
+        """
+        new_org = make_org({u'name': gen_alphanumeric()})
+        new_cv = make_content_view({u'organization-id': new_org['id']})
+        ContentView.publish({u'id': new_cv['id']})
+        result = ContentView.info({u'id': new_cv['id']})
+        env = result.stdout['lifecycle-environments'][0]
+
+        result = ContentView.remove({
+            u'id': new_cv['id'],
+            u'environment-ids': env['id'],
+            u'organization-id': new_org['id'],
+        })
+        self.assertEqual(result.return_code, 0)
+        result = ContentView.info({u'id': new_cv['id']})
+        self.assertFalse(result.stdout['lifecycle-environments'])
```

Comment:
Yes that is true, but when reading the test it may confuse if you are trying to check if that return value is True or False or what if you are expecting the list to not have any element on it. This is just a matter of meaning and we avoid using assertTrue and assertFalse in favor of being more explicit.

When reading self.assertEqual(len(...), 0) you don't need to think much to figure out that you are expecting no items and ... is a list. But when reading self.assertFalse(...) you have to check if that is a boolean result, and if not you have to know that it is a list and should not contain any item on it.